pkg/shell/nav.tsx

```diff
@@ -210,7 +258,18 @@ function FormattedText({ keyword, term }) {
     );
 }
 
-export function CockpitNavItem(props) {
+export function CockpitNavItem(props : {
+    name: string;
```

Comment:
FTR, I applaud replacing these PropTypes (which are only a runtime thing, and easy to ignore) with build-time types! :clap:

Reply:
We should have both, but we don't really check them at run time, and they are mostly wrong.

Reply:
I thought we did in the /devel scenario --- failing on React warnings is the whole reason why we have this (okay, also coverage)